src/components/Atoms/Action/Button/Button.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import style from './Button.module.css';
+
+const cx = classNames.bind(style);
+
+interface IButtonProps {
+  variant: string;
+  children: any;
```

Comment:
`children` is of type `ReactNode` wich is exported from `React`

Reply:
I changed this :)